Utilities/Dockers/RTG/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM ubuntu:22.10
+RUN apt update
+RUN apt install -y openjdk-11-jre ant git unzip python3 python3-pip
+
+# Get RTG files for install
+RUN git clone https://github.com/RealTimeGenomics/rtg-tools.git
+WORKDIR rtg-tools
+RUN ant zip-nojre
+WORKDIR dist
+RUN unzip *.zip 
+
+# Install RTG
+WORKDIR rtg-tools-3.12.1-32d4c2d2
```

Comment:
this feels tied to a specific version, will it break if rtg updates?

Reply:
Yeah it is a bit version specific. Although technically it won't "break" with an RTG release, since the docker build doesn't depend on any new releases once created, it would be nice to be able to rerun this when there's a new release to get an update. I couldn't find anything with using wildcards in `WORKDIR` statements… Do you have any ideas on getting around this? Something akin to `WORKDIR rtg-tools-*` ?

Reply:
I just checked, and it seems like the symbolic link doesn't work as expected, but `WORKDIR rtg-tools-*` does seem to work, so I changed to that.